l10n_fi_edicode/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```

Comment:
@jarmokortetjarvi What do you think about changing the license to LGPL-3? This is a very low-level module that might be required by various proprietary modules as well.

Reply:
I don't see much damage done there. The point of current l10n_fi modules are to provide a common fields for further development rather than to add any actual functionality.

Also: yay for adding identifier in name get and search - we actually just talked about making a PR for adding them, as customers often provide their operator code rather than operator name.

Reply:
So, what should I do? and `__manifest__.py` also contain some text about license.

Reply:
Please change it to LGPL

Reply:
done